src/Summoner/Template.hs

```diff
@@ -109,66 +109,96 @@ createStackTemplate ProjectData{..} = Dir (toString repo) $
         $endLine
         |]
 
+    createWarnings :: Text
+    createWarnings = case warnings of
+        [] -> defaultWarnings
+        xs -> "ghc-options:        " <> T.intercalate "\n                    " xs
+
+    defaultWarnings :: Text
+    defaultWarnings =
+        [text|
+        ghc-options:        -Wall
+                            -threaded
+                            -rtsopts
+                            -with-rtsopts=-N
```

Comment:
`-threaded` `-rtsopts` and `-with-rtsopts` should be in all of the `ghc-options` fields except library stanza no matter what.